leftwm/src/bin/leftwm-command.rs

```diff
@@ -49,12 +49,15 @@ async fn main() -> Result<()> {
         Available Commands:
 
         Commands without arguments:
-        
+
         UnloadTheme
         SoftReload
         ToggleFullScreen
+        ToggleSticky
         SwapScreens
         MoveWindowToLastWorkspace
+        MoveWindowToNextWorkspace
+        MoveWindowToPreviousWorkspace
```

Comment:
Thanks for adding these, I was thinking recently that I somehow miss some commands, but then thought we might just have disuccesd them but not implemented yet...